src/api/app/components/sourcediff_component.html.haml

```diff
@@ -1,6 +1,6 @@
 .row
   .col
-    - if @action.diff_not_cached({ diff_to_superseded: @diff_to_superseded })
+    - if diff_not_cached
```

Comment:
Which are the "unnecessary backend calls" being removed?

Reply:
`@action.diff_not_cached` was being executed three times: once in the `cache_diff_data` before action, once in `sourcedoff_component`, and a third time via an AJAX request triggered for `request_action_changes` action on diff load. It is now only executed in the before action. Additionally, the AJAX request no longer triggers automatically on page load.